common/static/js/vendor/jquery.leanModal.js

```diff
@@ -0,0 +1,61 @@
+// leanModal v1.1 by Ray Stone - http://finelysliced.com.au
```

Comment:
It would have to be included in one of the pipeline bundles. For example, here's where they are defined in the LMS:

https://github.com/edx/edx-platform/blob/master/lms/envs/common.py#L1455

It is probably too complicated to determine which bundle this should go in based upon its usages, so given how small it is I think we should let this go as is.